internal/character/qingque/trace.go

```diff
@@ -0,0 +1,65 @@
+package qingque
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	A2 key.Modifier = "qingque-a2"
+	A6 key.Modifier = "qingque-a6"
+)
+
+// A2:
+// 	Restores 1 Skill Point when using the Skill. This effect can only be triggered 1 time per battle.
+// A4:
+//	Using the Skill increases DMG Boost effect of attacks by an extra 10%.
+// A6:
+//	Qingque's SPD increases by 10% for 1 turn after using the Enhanced Basic ATK.
+
+func init() {
+	modifier.Register(A2, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
```

Comment:
this is not registered as a buff in game (from what I can tell, they dont even use a modifier)